docs/docs/en/architecture/configuration.md

```diff
@@ -185,19 +185,21 @@ Currently, common.properties mainly configures Hadoop,s3a related configurations
 |Parameters | Default value| Description|
 |--|--|--|
 data.basedir.path|/tmp/dolphinscheduler| local directory used to store temp files
-resource.storage.type|NONE| type of resource files: HDFS, S3, NONE
-resource.upload.path|/dolphinscheduler| storage path of resource files
+resource.view.suffixs| txt,log,sh,conf,cfg,py,java,sql,hql,xml,properties| file types supported by resource center
+resource.storage.type| NONE| type of resource files: HDFS, S3, NONE
+resource.storage.upload.base.path| /dolphinscheduler| storage path of resource files
+resource.aws.access.key.id| minioadmin| access key id of S3
+resource.aws.secret.access.key| minioadmin| secret access key of S3
+resource.aws.region|us-east-1| region of S3
+resource.aws.s3.bucket.name| dolphinscheduler-test| bucket name of S3
+resource.aws.s3.endpoint| http://minio:9000| endpoint of S3
+resource.hdfs.root.user| hdfs| configure users with corresponding permissions if storage type is HDFS
+resource.hdfs.fs.defaultFS| hdfs://mycluster:8020|If resource.storage.type=S3, then the request url would be similar to 's3a://dolphinscheduler'. Otherwise if resource.storage.type=HDFS and hadoop supports HA, copy core-site.xml and hdfs-site.xml into 'conf' directory
```

Comment:
We should add a space between each ```|``` to keep the format consistent.

Reply:
The modified